README.md

```diff
@@ -147,6 +147,7 @@ Broadcasting | +
 Periodical Timers | -
 Disconnect remote clients | -
 
+Anycable Rails contains optional compatibility checks. In order to turn dynamic checks on you need to put `require "anycable/rails/compatibility"` into your `development` and `test` environments. After that `Anycable::CompatibilityError` would be thrown when your code tries to do use any of the incompatible features. Do not try to use it on production!
```

Comment:
Let's remove this docs for now. There gonna be a separate repo for docs soon

Reply:
done